inputfiles/removedTypes.jsonc

```diff
@@ -705,6 +705,20 @@
                     }
                 }
             },
+            "RTCPeerConnectionIceEventInit": {
+                "members": {
+                    "member": {
+                        "url": null // Safari +12 only
+                    }
+                }
+            },
+            "FetchEventInit": {
```

Comment:
This file tries to make things alphabetically ordered, please move this above. Same for the event init.